tools/importPlanGoals.js

```diff
@@ -163,15 +156,12 @@ export default async function importGoals(file, region) {
               tp.goalId = goalId;
             }
           });
-          currentGranteeId = cleanGrantees.find((g) => g.name === currentGrantee).id;
           for await (const grant of currentGrants) {
-            const fullGrant = { number: grant.trim(), granteeId: currentGranteeId };
-            if (!cleanGrants.some((e) => e.granteeId === fullGrant.granteeId
-                            && e.number === fullGrant.number)) {
-              cleanGrants.push(fullGrant);
-            }
-            const [dbGrant] = await Grant.findOrCreate({ where: { ...fullGrant, regionId } });
+            const fullGrant = { number: grant.trim(), regionId };
+            const dbGrant = await Grant.findOne({ where: { ...fullGrant }, attributes: ['id', 'granteeId'] });
```

Comment:
Do we need to handle the grant/grantee not existing in the DB since we no longer create grants/grantees in the import?

Reply:
I think here we would want to fail if the grantee is not in the database - assuming that all Grant and Grantee data would be already present in the db since we pulled it from HSES